### PR TITLE
Blog post about support for multiple authors in Jekyll (to be posted after the other PR is merged :))

### PR DESCRIPTION
**Pull request by Fgerthoffert**

### CODE REVIEW

File: _posts/francois_gerthoffert/2016-12-09-Adding-Support-For-Multiple-Authors-in-Jekyll.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+authors:
+   - francois_gerthoffert
+   - jared_baker
+~~~
+
+## Listing Authors
+
+To list authors, the system will first count the size of the "author" and "authors" variable declared in the blog post, returning 0 if the variable does not exists or is empty.
+
+~~~
```
Comment (btiernay) [inline _posts/francois_gerthoffert/2016-12-09-Adding-Support-For-Multiple-Authors-in-Jekyll.md:42]: Does jekyll not support language specific code formatter?

File: _posts/francois_gerthoffert/2016-12-09-Adding-Support-For-Multiple-Authors-in-Jekyll.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+
+OICR being a research institute, a portion of the team, in particular those of us more on the science side, is used to [Academic Publishing](https://en.wikipedia.org/wiki/Academic_publishing) and its related conventions such as [Academic Authorship](https://en.wikipedia.org/wiki/Academic_authorship). It was not long before we were asked for supporting multiple authors in a blog post.
+
+This blog post will detail the solution we implemented to add this feature into our Jekyll implementation of the blog hosted on [GitHub Pages](https://pages.github.com/).
+
+## Jekyll & Liquid
+
+The blog is built on [Jekyll](http://jekyllrb.com/), which uses [Liquid](https://shopify.github.io/liquid/) for its template engine. Liquid provides basic operators, types and a set of filters to implement logic into templates.
+
+## Declaring Authors in a blog post
```
Comment (btiernay) [inline _posts/francois_gerthoffert/2016-12-09-Adding-Support-For-Multiple-Authors-in-Jekyll.md]: Inconsistent capitalization

File: _posts/francois_gerthoffert/2016-12-09-Adding-Support-For-Multiple-Authors-in-Jekyll.md [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+ </p>
+(% else if authorCount > 0 %)
+ (% assign author = site.data.authors[post.author] %)
+ <p class="post-author">By: <a href="{{site.baseurl}}/blog/category/{{post.author}}"><strong>{{ author.name }}</strong></a></p>
+(% endif %)
+
+# Note: Replace () with {} in the code above, for it to be understood by Liquid
+~~~
+
+## Listing Blog Posts for a particular Author
```
Comment (btiernay) [inline _posts/francois_gerthoffert/2016-12-09-Adding-Support-For-Multiple-Authors-in-Jekyll.md]: Inconsistent capitalization

File: _posts/francois_gerthoffert/2016-12-09-Adding-Support-For-Multiple-Authors-in-Jekyll.md (hunk 2 above)
Comment (Fgerthoffert) [reply _posts/francois_gerthoffert/2016-12-09-Adding-Support-For-Multiple-Authors-in-Jekyll.md]: 👍

File: _posts/francois_gerthoffert/2016-12-09-Adding-Support-For-Multiple-Authors-in-Jekyll.md (hunk 3 above)
Comment (Fgerthoffert) [reply _posts/francois_gerthoffert/2016-12-09-Adding-Support-For-Multiple-Authors-in-Jekyll.md]: 👍

File: _posts/francois_gerthoffert/2016-12-09-Adding-Support-For-Multiple-Authors-in-Jekyll.md (hunk 1 above)
Comment (Fgerthoffert) [reply _posts/francois_gerthoffert/2016-12-09-Adding-Support-For-Multiple-Authors-in-Jekyll.md:42]: Didn't look further than adding the code section and seeing {% being interpreted by jekyll

**Review by btiernay — APPROVED** [review]

**Comment by btiernay** [comment]
👍

**Pull request merged**